README.md

```diff
@@ -1,2 +1,9 @@
 # smove
-Package manager for Move in Substrate
+Package manager for Move language in Substrate.
+
+## Installation
+
+Install it from source with cargo by running the following:
```

Comment:
Later on, we can even provide some released binaries, but the preferred way right now is to install it from source